pixi.toml

```diff
@@ -72,41 +78,61 @@ rerun-web-release = { cmd = "cargo run --package rerun-cli --no-default-features
 # (this looks heavy but takes typically below 0.1s!)
 rerun-build-web-release = "rustup target add wasm32-unknown-unknown && cargo run -p re_build_web_viewer -- --release -g"
 
-
 build-examples = "cargo run -q --locked -p re_build_examples --"
 
+# Code formatting for all languages.
+format = { depends_on = ["py-fmt", "cpp-fmt", "toml-fmt", "misc-fmt"] }
+fmt = { depends_on = ["format"] }
+
 # Assorted linting tasks
 fast-lint = "python scripts/fast_lint.py"
 lint-codegen = "cargo run --package re_types_builder -- --check"
 # TODO(jleibs): implement lint-cpp-all
-lint-cpp-files = "clang-format --dry-run -Werror"
 lint-rerun = "python scripts/lint.py"
 lint-rs-files = "rustfmt --edition 2021 --check"
 lint-rs-all = "cargo fmt --check"
-lint-py-fmt-check = "ruff format --check --config rerun_py/pyproject.toml"
-lint-py-blackdoc = "blackdoc --check"
-lint-py-mypy = "mypy --install-types --non-interactive --no-warn-unused-ignore"
-lint-py-ruff = "ruff format --check --config rerun_py/pyproject.toml"
-lint-taplo = "taplo fmt --check --diff"
+
 lint-typos = "typos"
 
+# Toml tasks
+toml-fmt-check = "taplo fmt --check --diff"
+toml-fmt = "taplo fmt"
+
+# Misc formatting tasks.
 misc-fmt = "prettier --write '**/*.{yml,yaml,js,css,html}'"
 misc-fmt-check = "prettier --check '**/*.{yml,yaml,js,css,html}'"
+
 ruff-fmt = "ruff format --config rerun_py/pyproject.toml ."
-ruff-fix = "ruff --fix --config rerun_py/pyproject.toml ."
-toml-fmt = "taplo fmt"
+ruff-fix = "ruff check --fix --config rerun_py/pyproject.toml ."
```

Comment:
Can we make commands "private" somehow? The user should focus on `py-fmt` I think, not this implementation detail. Maybe just move it to later in the file?

(I think commands should be high level, and not focus on how they are implemented, i.e. "ruff")

Reply:
> I think commands should be high level

agreed!
Not aware of a way to hide commands - we could also just make the py command combine these.